TUTORIAL.md

```
@@ -304,7 +304,7 @@ func main() {
 				return
 			} else if ev.Key() == tcell.KeyCtrlL {
 				s.Sync()
-			} else if ev.Rune() == 'C' || ev.Rune() == 'c' {
+			} else if ev.Str() == "C" || ev.Str() == "c" {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)